.rspec

```diff
@@ -1,4 +1,3 @@
 --format documentation
 --color
 --require spec_helper
---pattern "{source/,spec/}**{,/*/**}/*_spec.rb"
```

Comment:
I was wondering what was happening when I tried to run specific files 👍🏼

Reply:
yeah luckily we have `rake spec` configured to run them all still